src/overlays/actors/ovl_En_Gamelupy/z_en_gamelupy.h

```diff
@@ -5,17 +5,25 @@
 
 struct EnGamelupy;
 
+#define ENGAMELUPY_POINTS 50
+
 typedef void (*EnGamelupyActionFunc)(struct EnGamelupy*, PlayState*);
 
+typedef enum {
+    /* 0 */ ENGAMELUPY_TYPE_GREEN,
+    /* 1 */ ENGAMELUPY_TYPE_BLUE,
+    /* 2 */ ENGAMELUPY_TYPE_MAX
+} EnGamelupyType;
+
 typedef struct EnGamelupy {
     /* 0x000 */ Actor actor;
     /* 0x144 */ ColliderCylinder collider;
     /* 0x190 */ EnGamelupyActionFunc actionFunc;
-    /* 0x194 */ s32 rupeeIndex;
-    /* 0x198 */ s16* unk_198;
-    /* 0x19C */ s16 unk_19C;
-    /* 0x19E */ s16 unk_19E;
-    /* 0x1A0 */ s16 unk_1A0;
+    /* 0x194 */ s32 type;
+    /* 0x198 */ s16* minigameScore; // Shared memory location with actor EnLiftNuts
```

Comment:
Just a minor nit to make this super clear. Feel free to push back; I just felt it was nice considering this concept literally does not appear in any other actor I think?
```suggestion
    /* 0x198 */ s16* minigameScore; // Pointer to shared memory location with actor EnLiftNuts
```